pydis_site/static/css/content/page.css

```diff
@@ -0,0 +1,16 @@
+.breadcrumb-section {
+    padding: 1rem;
+}
+
+i.has-icon-padding {
+    padding: 0 10px 25px 0;
+}
+
+pre {
+    /*
+     * Style it the same as the <code> tag, since highlight.js does not style
+     * backgrounds of <pre> tags but bulma does, resulting in a weird off-white
+     * border.
+     */
+    background-color: #282c34;
```

Comment:
It would not, since that would result in a white `<pre>` background and a cramped `<code>` block.

![image](https://user-images.githubusercontent.com/41782385/112306191-82ca7e80-8cda-11eb-8966-e987b081bdec.png)

As a workaround to avoid hardcoding, I reduced the padding of `<pre>` and increased the `<code>` padding so it looks the same, but doesn't rely on specifying a background color.